frontend/package.json

```diff
@@ -13,13 +13,13 @@
         "test-coverage": "vitest run --coverage"
     },
     "dependencies": {
-        "@bcgov-nr/nr-theme": "^1.4.2",
+        "@bcgov-nr/nr-theme": "^1.5.0",
         "@carbon/icons-vue": "^10.72.1",
         "aws-amplify": "^5.3.6",
         "axios": "0.26.1",
         "bootstrap": "^5.2.3",
         "fam-api": "file:../client-code-gen/gen",
-        "primevue": "^3.27.0",
+        "primevue": "3.34.0",
```

Comment:
The latest primevue version v3.36.0 has some style change that we need to adjust. For now we can stick with v3.34.0.